api/lib/mno_enterprise/concerns/controllers/jpi/v1/impac/alerts_controller.rb

```diff
@@ -22,10 +22,10 @@ def create
 
     authorize! :manage_alert, kpi_alert
 
-    if (@alert = current_user.alerts.create(kpi_alert.attributes))
+    if (@alert = kpi_alert.save(kpi_alert.attributes))
```

Comment:
super-cool if we can do that and flag the corresponding version of mnohub as a dependency..